src/main.js

```diff
@@ -157,12 +166,66 @@ const createWindow = async () => {
       await shutdownPythonProcess();
     }
   });
+
+  // Setup handlers & defaults to manage downloads.
+  // Specifically, invest sample data downloads
+  let downloadDir;
+  let downloadLength;
+  const downloadQueue = [];
+  ipcMain.on('download-url', async (event, urlArray, directory) => {
+    logger.debug(`${urlArray}`);
+    downloadDir = directory;
```

Comment:
Is `downloadDir` used at all here?

Reply:
Yeah, it's not obvious, but it's assigned here and used in the 'will-download' listener.

Reply:
Ahhh, yea, I see it now. Thanks.